odoo/addons/base/views/ir_attachment_views.xml

```diff
@@ -36,7 +36,7 @@
                             <field name="description" nolabel="1"/>
                         </group>
                         <group groups="base.group_no_one" string="Indexed Content" colspan="4">
-                            <field name="index_content" nolabel="1"/>
+                            <field style="word-wrap: anywhere;" name="index_content" nolabel="1"/>
```

Comment:
don't we have already a class for that ? (BS or odooish class) ?

Reply:
indeed ` class="text-break"` from BS would be a better fit